src/index.ts

```diff
@@ -1,7 +1,7 @@
-import { auth } from "./auth";
-import { getState } from "./get-state";
-import { StrategyOptions, State } from "./types";
-import { VERSION } from "./version";
+import { auth } from "./auth.js";
```

Comment:
Why do you import the .js file?

Reply:
It's so the output has a proper ESM import.
You have to have a proper full relative path in imports

Reply:
I didnt need to do that in my PR. Doesnt typescript take care of that + we bundle anyway?

Reply:
The imports will work fine in CJS, but not in ESM output.
If you try to import the `dist-src` it will fail because the imports aren't proper

TypeScript doesn't add any file extensions to the imports.

Reply:
But if it is bundled we wont import that files?